exe/kubernetes-run

```diff
@@ -23,10 +30,12 @@ logger = KubernetesDeploy::FormattedLogger.build(namespace, context)
 runner = KubernetesDeploy::RunnerTask.new(
   namespace: namespace,
   context: context,
-  logger: logger
+  logger: logger,
+  max_watch_seconds: max_watch_seconds
```

Comment:
Here its to keep parity with [`kubernetes-deploy`](https://github.com/Shopify/kubernetes-deploy/blob/master/exe/kubernetes-deploy#L69). 

I added `max_watch_seconds` to new in `kubernetes-deploy` since is used like 3 levels deep in methods call from `run` and I didn't want to just pass it down when it could be an instance var, and it seemed like instance vars should be set by new. Can't speak to `verify_result `, but the split is probably an oversight / wasn't noticed till now.

Reply:
I don't think this a super big deal since an instance will rarely (if ever) get `run` multiple times in practice. I can't say I remember why I put each argument where I did. For now, I think consistency with the other commands is the most important thing, and if we want to change them in the future (a breaking change), we should do it for all three commands at once.